anvil-rstudio-base/README.md

```diff
@@ -1,19 +1,82 @@
-# RStudio Docker Image
+| RStudio Version | Docker Image URL | Date Updated | Questions or Feedback |
+| --- | --- | --- | --- |
+| 0.0.2 | us.gcr.io/anvil-gcr-public/anvil-rstudio-base:0.0.2 | 11/26/2019 | Contact the [Interactive Analysis Team](mailto:workbench-interactive-analysis@broadinstitute.org) |
```

Comment:
what's the purpose of this section? is this supposed to be a changelog of all images? also, is rstudio version here correct? (cc @rtitle )

Reply:
The purpose is to give the reader quick information about the Docker. The RStudio Version should read "RStudio Docker Version". I will change

Reply:
so only latest image goes here?

Reply:
I'm asking because we also have https://github.com/anvilproject/anvil-docker/blob/master/anvil-rstudio-base/CHANGELOG.md#002---11262016

Reply:
Yes

Reply:
The table at the top is not meant to be version history or changelog. Just the most the recent version and url.